### ISSUE TITLE
每天3分钟操作系统修炼秘籍（17）：进程间通信(3)：套接字 | 骏马金龙

### ISSUE THREAD

**Issue by malongshuai** [malongshuai/malongshuai.github.io#76]
https://www.junmajinlong.com/os/IPC_socket/ 

 每天3分钟操作系统修炼秘籍（17）：进程间通信(3)：套接字

**Comment by tan-siying** [comment]
那是不是可以理解为 socket 和 上篇的管道类似，也是两个 fd 加上一段内核的内存--socket buffer

**Comment by malongshuai** [comment]
> @tan-siying
> 那是不是可以理解为 socket 和 上篇的管道类似，也是两个 fd 加上一段内核的内存--socket buffer

是的。如果没记错的话，据《Linux系统编程手册》还是某本比较权威的书介绍，曾有操作系统的管道就是通过Unix Domain Socket来实现的。
